xds/src/main/java/io/grpc/xds/WrrLocalityLoadBalancer.java

```diff
@@ -80,17 +100,17 @@ public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
   @Override
   public void handleNameResolutionError(Status error) {
     logger.log(XdsLogLevel.WARNING, "Received name resolution error: {0}", error);
-    if (childLb != null) {
-      childLb.handleNameResolutionError(error);
+    if (switchLb != null) {
```

Comment:
graceful switch handles the case where you receive an error before calling switchTo(), so you don't need the else here. https://github.com/grpc/grpc-java/blob/f2348b0157cae21074a48d2c0428c9750fc51b2f/core/src/main/java/io/grpc/util/GracefulSwitchLoadBalancer.java#L59-L75